pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -140,6 +140,12 @@ func (admitter *VMsAdmitter) authorizeVirtualMachineSpec(ar *v1beta1.AdmissionRe
 		}
 	}
 
+	if len(causes) > 0 {
+		return causes, nil
+	}
+
+	causes = validateVMStartAllowed(vm)
```

Comment:
You mean like this right?
`if len(causes) > 0 {
		return causes, nil
	}`

Reply:
I think it's redundant, if I understood correctly, that's how it would look like afterwards
```
causes = validateVMStartAllowed(vm)

	if len(causes) > 0 {
		return causes, nil
	}
	
	return causes, nil
```